CHANGELOG.md

```diff
@@ -94,7 +94,7 @@ In this document, all remarkable changes are listed. Not mentioned are smaller c
 
 ## 0.4.2
 
-- users are now allowed to save `None` buffers for a `GGRSRequest::SaveRequest`. This allows users to keep their own state history and load/save more efficiently
+- users are now allowed to save `None` buffers for a `GgrsRequest::SaveRequest`. This allows users to keep their own state history and load/save more efficiently
```

Comment:
Maybe skip the changelog, since we can't change naming for released versions?

Reply:
I think this should remain as is. It's more of a historical listing than anything else :)